src/code/views/link-relation-view.coffee

```diff
@@ -167,21 +167,30 @@ module.exports = LinkRelationView = React.createClass
           @renderScalarPulldown(@state.selectedScalar)
         )
       )
-      (div {className: 'graph', id:'relation-graph'},
-        (Graph {xAxis: source, yAxis: target, link: @props.link, graphStore: @props.graphStore})
-      )
-      if @props.link.relation.isCustomRelationship and not @props.link.relation.customData?
-        (div {className: 'graph-hint'},
-          (span {}, "#{tr "~NODE-RELATION-EDIT.CUSTOM_HINT"} ")
+      (div {className: 'bottom'},
+        (div {className: 'graph', id:'relation-graph'},
+          (Graph
+            xAxis: source
+            yAxis: target
+            link: @props.link
+            graphStore: @props.graphStore
+          )
         )
-      (div {className: 'fixed-bottom'},
+      )
+
+      (div {className: 'bottom'},
```

Comment:
The change from `fixed-bottom` to `bottom` makes it so that the graph and text box move when the second pull-down is added. Previously, this was adjusted so that those elements remained fixed in place at the bottom of the panel.

(I am to blame for overloading various variations on the classes named "bottom." It's the effect of making one incremental change after another and not cleaning them up...)